src/utils/generateNavLink.tsx

```diff
@@ -0,0 +1,11 @@
+import { NavLink } from 'react-router-dom';
+import { Person } from '../types';
+import { setPersonLinkClassName } from './classNamesSetter';
+
+export const generateNavLink = (person: Person) => {
```

Comment:
It would be better to use object destructuring for accessing object properties. In this case, you can extract 'slug' and 'name' from the 'person' object, which makes the code cleaner and easier to read.